joplin/templates/wagtailadmin/edit_handlers/tabbed_interface.html

```
@@ -2,6 +2,7 @@
 {% load wagtailadmin_tags %}
 <ul class="tab-nav merged" role="tablist">
     {% for child in self.children %}
+        {% if not child.hide_panel %}
         <li class="{{ child.classes|join:" " }} {% if forloop.first %}active{% endif %}" role="tab" aria-controls="tab-{{ child.heading|cautious_slugify }}">
```

by default show the seo & settings tab unless we've set it to hide